torch/fx/experimental/symbolic_shapes.py

```diff
@@ -409,7 +409,11 @@ class DimDynamic(Enum):
 # eager code with StrictMinMaxConstraint will keep working in the future!
 
 @dataclass(frozen=True)
-class StrictMinMaxConstraint:
+class Constraint:
+    warn_only : bool
```

Comment:
yo dawg, run lint lol

Reply:
This is linted, I run lint every push.

Reply:
you're failing lint on your most recent push https://github.com/pytorch/pytorch/actions/runs/4664408930/jobs/8256809135